src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -1269,6 +1269,24 @@ public async Task<OrganizationUser> AcceptUserAsync(string orgIdentifier, User u
         return await AcceptUserAsync(orgUser, user, userService);
     }
 
+    public async Task<OrganizationUser> AcceptUserAsync(Guid organizationId, User user, IUserService userService)
```

Comment:
It doesn't appear this method is added to the interface, if it's not needed there it should be `private` or `internal` if used for testing and it has no need to take the service as a parameter if it exists as a field to this class.

Reply:
Fixed by adding to interface